scripts/trace_flowcell.py

```
@@ -0,0 +1,118 @@
+                                             "workflow. It can be used to track executions, e.g: "
+                                             "python scripts/trace_runfolder.py --tag 150605_M00485_0183_000000000-ABGT6_testbio14 | xargs -n1 st2 execution get")
+    parser.add_argument('--flowcell', required=True)
+    parser.add_argument('--api_base_url', default="https://localhost:4041/api/v1")
+    parser.add_argument('--workflow', default="workflow")
+    args = parser.parse_args()
+
+    api_base_url = args.api_base_url
+    workflow_name = args.workflow
+# Load config
+#with open("/opt/stackstorm/packs/snpseq_packs/config.yaml", 'r') as ymlfile:
+    with open("/home/riker752-local/projects/snpseq_packs/config.yaml", 'r') as ymlfile:
+        cfg = yaml.load(ymlfile)
+
+# Get hosts from config
+    hosts = cfg['runfolder_svc_urls']
+
+# Print header
+    print "\tstatus\trunfolder_link"
+
+    search_term = args.flowcell
+    hitcount = 0
+    choice = 0
+    folders = {}
+    for host in hosts:
+        url_base = '/'.join(host.split('/')[:-1])
+        result = requests.get("{}?state=*".format(url_base))
+        result_json = json.loads(result.text)
+        all_runfolders = result_json["runfolders"]
+        for runfolder in all_runfolders:
+            link = runfolder["link"]
+            state = runfolder["state"]
+            if search_term in link and not "rchive" in link and not "biotank" in link:
+                hitcount += 1
+                dirs = link.split('/')
+                folders[hitcount] = dirs[-1]
+                print "{}\t{}\t{}".format(hitcount, state, folders[hitcount])
+
+        if hitcount > 1:
+            choice = int(raw_input("Which runfolder to trace: "))
+        else:
+            choice = 1
+      
+        print "Will trace " + folders[choice]
+
+    # Try to load access token from environment - fall back if not available
+        try:
+            access_token = os.environ['ST2_AUTH_TOKEN']
```

it would be a bit more user-friendly to check for this early in the script in order to fail early